src/Utilities/ToolLocationHelper.cs

```diff
@@ -985,7 +984,19 @@ string registryRoot
                         if (extensionSdk.SDKType == SDKType.Framework || extensionSdk.SDKType == SDKType.Platform)
                         {
                             // We don't want to attempt to gather ApiContract references if the framework isn't explicitly marked as Framework/Platform
-                            extensionSdkReferences = GetApiContractReferences(extensionSdk.ApiContracts, targetSdkPath);
+                            string platformKey = TargetPlatformSDK.GetSdkKey(targetSdkIdentifier, targetSdkVersion);
+                            PlatformManifest manifest;
+                            if (TryGetPlatformManifest(matchingSdk, platformKey, out manifest))
+                            {
+                                if (manifest.VersionedContent)
+                                {
+                                    extensionSdkReferences = GetApiContractReferences(extensionSdk.ApiContracts, matchingSdk.Path, manifest.PlatformVersion);
+                                }
+                                else
+                                {
+                                    extensionSdkReferences = GetApiContractReferences(extensionSdk.ApiContracts, matchingSdk.Path);
+                                }
+                            }
```

Comment:
It seems like that `else` should move up one level and it would only add the parameter if `TryGetPlatformManifest` returned `true` and `VersionedContent` was `true`, otherwise assume it's not?